src/runtime/virtcontainers/sandbox.go

```diff
@@ -2265,22 +2265,24 @@ func (s *Sandbox) resourceControllerDelete() error {
 		return nil
 	}
 
-	sandboxController, err := resCtrl.LoadResourceController(s.state.SandboxCgroupPath)
+	sandboxController, err := resCtrl.LoadResourceController(s.state.SandboxCgroupPath, s.config.SandboxCgroupOnly)
 	if err != nil {
 		return err
 	}
 
 	resCtrlParent := sandboxController.Parent()
-	if err := sandboxController.MoveTo(resCtrlParent); err != nil {
-		return err
+	if resCtrlParent != "." {
```

Comment:
This is an unrelated fix right? This might make more sense as a separate commit with a good description.

Reply:
This should be related, when i set sandbox_cgroup_only=true in /etc/kata-containers/configuration.toml, `sandboxController` will use `cgroupsv2.LoadSystemd` as the manager, but the `sandboxController.Parent()` return `.`, that cause `sandboxController.MoveTo(resCtrlParent)` return `cgroupsv2.ErrInvalidGroupPath
` error.

My solution is to skip it, but I'm not sure if that is a good solution.